Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_-_Host_Advanced_Hunting.yml

```diff
@@ -985,17 +1101,103 @@ tasks:
     view: |-
       {
         "position": {
-          "x": -570,
-          "y": 890
+          "x": -90,
+          "y": 1090
+        }
+      }
+  '89':
+    id: '89'
+    taskid: 0a3ba1a2-67eb-4d83-8e17-962c519073d4
+    type: condition
+    task:
+      id: 0a3ba1a2-67eb-4d83-8e17-962c519073d4
+      version: -1
+      name: Is there a hash?
+      description: Is there a hash?
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#default#':
+      - '40'
+      yes:
+      - '86'
+    separatecontext: false
+    conditions:
+    - label: yes
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.FileSha1
```

Comment:
It's because the next task only supports sha1 😮‍💨 (based on the API).
![image](https://user-images.githubusercontent.com/88268646/179696514-026e93fc-0ddc-4521-8cf2-0e09feb5ff98.png)